.github/workflows/post-merge-public-api.yml

```diff
@@ -160,7 +160,53 @@ jobs:
                   github-token: ${{ secrets.GITHUB_TOKEN }}
                   linear-api-key: ${{ secrets.LINEAR_API_KEY }}
                   slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
-                  service-name: Public API
+                  service-name: Public API (East US)
+
+    deploy_public_api_to_prod_central_india:
+        if: needs.check_should_deploy.outputs.should_deploy == 'true'
+        name: Deploy Public API to prod (Central India)
+        runs-on: ubuntu-latest
+        needs:
+            - build
+            - check_should_deploy
+            - deploy_public_api_to_prod_east_us
+        environment:
+            name: prod-public
+            url: ${{ vars.URL_PROD }}
+        steps:
+            - name: Checkout source code
+              uses: actions/checkout@v4.1.7
+              with:
+                  ref: ${{ github.event.pull_request.head.sha }}
+            - uses: ./.github/actions/deploy-to-env
+              id: deploy
+              with:
+                  environment: prod
+                  full-environment: prod-public
+                  connection-url: ${{ vars.DB_CONNECTION_URL_PROD }}
+                  web-app-name: ${{ vars.AZURE_WEB_APP_NAME_PUBLIC_PROD_INDIA }}
+                  resource-group: ${{ vars.AZURE_RESOURCE_GROUP_PROD_INDIA }}
+                  azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+                  azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+                  azure-subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+                  linear-api-key: ${{ secrets.LINEAR_API_KEY }}
+                  slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
+                  service-name: Public API (Central India)
+
+    delete_hotfix_branch:
```

Comment:
Moving this to a separate job that depends upon the deployment jobs prevents deleting the hotfix branch until all deployments have succeeded.